core/command.h

```diff
@@ -98,6 +99,18 @@ int main (int cmdline_argc, char** cmdline_argv)
     ::MR::GUI::App app (cmdline_argc, cmdline_argv);
 #endif
     ::MR::App::parse ();
+
+    //ENVVAR name: MRTRIX_CLI_PARSE_ONLY
+    //ENVVAR Set the command to parse the provided inputs and then quit
+    //ENVVAR if it is 1. This can be used in the CI of wrapping code,
+    //ENVVAR such as the automatically generated Pydra interfaces
+    char* parse_only = std::getenv("MRTRIX_CLI_PARSE_ONLY");
+    if (parse_only && ::MR::to<int>(parse_only)) {
```

Comment:
```suggestion
    if (parse_only && ::MR::to<bool>(parse_only)) {
```

https://github.com/MRtrix3/mrtrix3/blob/3.0.4/core/mrtrix.h#L285-L293